src/main/java/com/sintern/service/OpenInternPositionService.java

```diff
@@ -1,12 +1,15 @@
 package com.sintern.service;
 
+import com.sintern.domain.Domain;
 import com.sintern.domain.DomainType;
 import com.sintern.domain.OpenInternPosition;
 import com.sintern.repository.OpenInternPositionRepository;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
+import java.util.ArrayList;
 import java.util.List;
+import java.util.stream.Collectors;
 
 @Service
 public class OpenInternPositionService {
```

Comment:
We should have an interface called **OpenInternPositionService** and a concrete implementation - **OpenInternPositionServiceImpl**.

Reply:
I understand, however, when I pulled the project from the branch that you specified, this is how the service was implemented. I only added the getDomains method.